src/modules/curations/itemCuration/reducer.ts

```diff
@@ -43,7 +60,47 @@ export function itemCurationReducer(state: ItemCurationState = INITIAL_STATE, ac
         },
         error: null
       }
+    }
+
+    case PUBLISH_THIRD_PARTY_ITEMS_SUCCESS: {
+      const { itemCurations } = action.payload
+      const oldItemCurations = state.data[action.payload.collectionId]
+      const mergedItemCurations = oldItemCurations ? [...oldItemCurations, ...itemCurations] : itemCurations
```

Comment:
Maybe it's best to also do `[...itemCurations]` to avoid having a pointer to the same array on the state

Reply:
updated!